lib/src/const_date_time.dart

```diff
@@ -195,4 +195,22 @@ class ConstDateTime implements DateTime {
   bool operator ==(Object other) {
     return dateTime == other;
   }
+
+  @override
+  dynamic noSuchMethod(Invocation invocation) {
+    // On the Web platform, the [DateTime.difference] method reads the `other._value` variable internally.
+    // ConstDateTime does not have a `_value` variable, which leads to errors when running
+    // Returning [millisecondsSinceEpoch] in this case will prevent the error from being reported.
+    //
+    // @see https://github.com/dart-lang/sdk/blob/56e581aa3415858bbaf22bee23be705a68f8a03e/sdk/lib/_internal/js_runtime/lib/core_patch.dart#L342
+    if (_kIsWeb && invocation.isGetter
```

Comment:
Should we strengthen this check by adding `&& !invocation.isSetter`? I believe final properties follow this pattern.